### DISCUSSION TITLE
manual/help and versioning output

### DISCUSSION THREAD

**Discussion by francescobianco** [francescobianco/shell-output-guidelines#14] [General]
which concerns to manual/help and versioning output

**Comment by Potherca** [comment]
Here, regarding `--help` output, I think it is worth noting [docopt](http://docopt.org/) and [Utility Argument Syntax](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html) from the POSIX standard.

Some further searching leads to the Stack Overflow question [_"Is there a "standard" format for command line/shell help text?"_](https://stackoverflow.com/questions/9725675/is-there-a-standard-format-for-command-line-shell-help-text) which has a lot of useful links to other options in various answers.
